docs/platform/object-storage/how-to-access-objects-with-linode-object-storage/index.md

```diff
@@ -0,0 +1,75 @@
+---
+author:
+  name: Linode Community
+  email: docs@linode.com
+description: "This guide shows you multiple ways to access your objects stored in Linode's Object Storage."
+og_description: "This guide shows you multiple ways to access your objects stored in Linode's Object Storage."
+keywords: ['object','storage','bucket']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2020-01-22
+modified: 2020-01-22
+modified_by:
+  name: Linode
+title: "How to Access Objects with Linode Object Storage"
+h1_title: "Accessing Objects with Linode Object Storage"
+contributor:
+  name: Linode
+---
+
+{{< note >}}
+[Linode Object Storage](/docs/platform/object-storage/) is now available to the general public in the Newark data center! Starting November 1, 2019, all customers with the Object Storage service enabled on their account will be billed. For more information, see our [Object Storage Pricing and Limitations](/docs/platform/object-storage/pricing-and-limitations/) guide.
+{{</ note >}}
+
+Object Storage gives each object a unique URL with which you can access your data. An object can be publicly accessible, or you can set it to be private and only visible to you. This makes Object Storage great for sharing and storing unstructured data like images, documents, archives, streaming media assets, and file backups, and the amount of data you store can range from small collections of files up to massive libraries of information.
+
+In this guide you will learn how to access the objects you have stored in Linode's Object Storage for:
```

Comment:
I don't think "for" is right here, I think "using" would be better.

Reply:
done